scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/layers/TransformerLayer.scala

```diff
@@ -27,7 +27,7 @@ import com.intel.analytics.bigdl.dllib.keras.Net
 import com.intel.analytics.bigdl.dllib.keras.autograd.{AutoGrad, Constant, Variable}
 import com.intel.analytics.bigdl.dllib.keras.layers.utils.KerasUtils
 import com.intel.analytics.bigdl.dllib.keras.models.Model
-import com.intel.analytics.bigdl.dllib.keras.Sequential._
+import com.intel.analytics.bigdl.dllib.keras.Sequential
```

Comment:
I think you need to be consistent:
```scala 
import com.intel.analytics.bigdl.dllib.keras.models.Model
import com.intel.analytics.bigdl.dllib.keras.models.Sequential
```
or
```scala 
import com.intel.analytics.bigdl.dllib.keras.Model
import com.intel.analytics.bigdl.dllib.keras.Sequential
```

You may do both by making one of them a subclass of the other.

Reply:
Ok. Will rename scala `dllib.keras.model` in another pr as it includes lots of files changed. This pr only rename scala `dllib.keras.sequential`